_aggregations/bucket/missing-bucket.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: Handling missing buckets
+parent: Composite
+grand_parent: Bucket aggregations
+great_grand_parent: Aggregations
+nav_order: 20
+---
+
+## Handling missing buckets
+
+By default, composite aggregations exclude documents that do not have a value for a particular source. However, you can choose to include these missing values by setting the `missing_bucket` parameter to `true` for the relevant source.
+
+## Syntax 
+
+The syntax for handling missing values in a composite aggregation requires you to include the `missing_bucket` parameter with a value of `true` within the relevant source definition, as shown in the following example syntax for the `sources` array.
+
+```json
+"sources": [
+  {
+    "NAME": {
+      "AGGREGATION": {
+        "field": "FIELD",
+        "missing_bucket": true
+      }
+    }
+  }
+]
+```
+{% include copy-curl.html %}
+
+---
+
+## Example
+
+For example, the following query groups documents by product name using a `terms` aggregation and includes a bucket for documents that do not have a product name specified: 
+
+```json
+GET /sales/_search
+{
+  "size": 0,
+  "aggs": {
+    "sales_by_day_product": {
+      "composite": {
+        "sources": [
+          {
+            "day": {
+              "date_histogram": {
+                "field": "timestamp",
+                "calendar_interval": "1d",
+                "order": "desc"
+              }
+            }
+          },
+          {
+            "product": {
+              "terms": {
+                "field": "product.keyword",
+                "order": "asc",
+                "missing_bucket": true
+              }
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+#### Example response
```

Comment:
In this response, it looks like the data you used to run query did not had missing values for a field as all the buckets look like the field values were present.